app/services/account_creation/device_profile/proofer.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module AccountCreation
+  module DeviceProfile
+    class Proofer < Proofing::LexisNexis::Ddp::Proofer
```

Comment:
It's a bit of a tricky arrangement with how the existing code is tightly-bound to "proofing". I think this approach works well enough, though I'd also wondered if we should try to make the DDP code moved/named somewhere more generic, or maybe we leave it as is and parameterize it in such a way that we can call it directly without needing additional code specific to authentication account creation.

Reply:
Yea. I was kind of waffling between both approaches. I would be open to moving the DDP code or having a generic DDP class.